src/components/game-board/board.tsx

```diff
@@ -16,6 +16,11 @@ export const MyContext = createContext("my-context");
 export const GameBoard = component$(() => {
   useStylesScoped$(styles);
 
+  const answer = Array.from({ length: 4 }, () => {
+    const randomColor = Math.floor(Math.random() * 6);
```

Comment:
if QWERTY_array is abstracted, the hardcoded `Math.random() * 6` should be `Math.random() * QWERTY_array.count()`